static/css/page-user.less

```diff
@@ -152,13 +152,19 @@ th.toggle-all {
     }
   }
 }
-
+/* stylelint-disable no-descending-specificity */
 .import-export {
   display: flex;
-  .import {
+  flex-wrap: wrap;
+  .import-export-readinglog {
```

Comment:
.. would be come `&__readinglog {`

This keeps the CSS linked, but the resulting CSS is much simpler in terms of specificity.